cmake/modules/SwiftSource.cmake

```diff
@@ -284,16 +291,16 @@ function(_compile_swift_files dependency_target_out_var_name)
       message(FATAL_ERROR "Don't know where to put the module files")
     endif()
 
+    list(APPEND swift_flags "-parse-as-library")
     if (NOT SWIFTFILE_EMIT_SIB)
       # Right now sib files seem to not be output when we emit a module. So
       # don't emit it.
       set(module_file "${module_dir}/${SWIFTFILE_MODULE_NAME}.swiftmodule")
       set(module_doc_file "${module_dir}/${SWIFTFILE_MODULE_NAME}.swiftdoc")
-      list(APPEND swift_flags
-          "-parse-as-library"
-          "-emit-module" "-emit-module-path" "${module_file}")
-    else()
-      list(APPEND swift_flags "-parse-as-library")
+      list(APPEND module_command
+          "-emit-module"
+          "-emit-module-path" "${module_file}"
+          "-o" "${module_file}")
```

Comment:
You shouldn't need both `-emit-module-path` and `-o` here. Either one should be fine.